config/lib/express.js

```diff
@@ -360,6 +374,7 @@ module.exports.initHelmetHeaders = function(app) {
           'fcm.googleapis.com',
           'maitreapp.co', // Signup waiting list feature
           'www.facebook.com',
+          'https://sentry.io',
```

Comment:
I think this should be just `sentry.io` without protocol.

Reply:
I always include protocol too, I'll keep it like this for now, should consider changing them all to be consistent, but at least having one different provokes the question until it's resolved ;)